esp/applications/common/ldap/ldap.yaml

```diff
@@ -20,4 +20,6 @@ ldap:
   workunitsBasedn: ou=workunits,ou=ecl
   ldapAdminSecretKey: "ldapadmincredskey"
   ldapAdminVaultId: ""
+  hpccAdminSecretKey: ""
```

Comment:
Would it work without the tag being defined?  Are these empty strings just here to be self documenting?

Reply:
If I added the first one it was probably because I wanted to guarantee the automatically generated esp.xml that is generated from this yaml to be identical to what existed in the past.  But it's probably not necessary for anything new. But as I hinted at, these can help the file/format be self documenting.

Reply:
Yes it would work without the tag. The behaviour in that case would be to not create the HPCC Admin user, forcing the Ops team to create one manually on the Domain Controller as they have in the past. The reason I put them in is, as you stated, to document what is available. To me this is way better than forcing the user to look in the manuals, manually type them into the override file with possible spelling errors.  I prefer to cut/paste/modify, but I am not sure if there is a precedent set by other components